report_qweb_pdf_watermark/models/report.py

```diff
@@ -17,73 +17,66 @@
     from PIL import PdfImagePlugin  # noqa: F401
 except ImportError:
     pass
+try:
+    from papersize import parse_couple, SIZES
+except ImportError:
+    raise
 from openerp import api, models, tools
 logger = getLogger(__name__)
 
 
 class Report(models.Model):
     _inherit = 'report'
 
-    @api.v7
-    def get_pdf(
-        self, cr, uid, ids, report_name, html=None, data=None, context=None
-    ):
-        # pylint: disable=R8110
-        # this override cannot be done in v8 api
-        result = super(Report, self).get_pdf(
-            cr, uid, ids, report_name, html=html, data=data,
-            context=context
-        )
-        report = self._get_report_from_name(cr, uid, report_name)
-        watermark = None
+    @api.multi
+    def _scale_watermark(self, report, image):
+        size = parse_couple(
+            SIZES[report.paperformat_id.format.lower()])
```

Comment:
as you pointed out in a comment before, this could fail. So handle this by logging an error and not scaling